src/App.js

```diff
@@ -1,106 +1,113 @@
-import React, { Component } from 'react';
-import uuid from 'uuid';
-import './App.css';
-import GameItem from './components/GameItem.js';
-import logo from './images/logo.png';
+import React, { Component } from "react";
+import uuid from "uuid";
+import "./App.css";
+import GameItem from "./components/GameItem.js";
+import logo from "./images/logo.png";
 
 class App extends Component {
   config = {
     itemTypes: {
       // type: spawn rate (weighting)
-      litter:  20,
-      rock:     5,
-      bush:     5,
-      flower:   5,
-      mushroom: 5,
+      litter: 20,
+      rock: 5,
+      bush: 5,
+      flower: 5,
+      mushroom: 5
     },
     spawnRate: 1.2, // Hz
     spawnRateRnd: 1.79, // randomization factor
     spawnHeight: 100, // height of item spawn area in pixels
     spawnFloor: 0, // offset from bottom of game "level" in pixels
-    itemLifetime: 10 * 1000, // 10 seconds (should be longer than CSS animation time)
-  }
+    itemLifetime: 10 * 1000 // 10 seconds (should be longer than CSS animation time)
+  };
 
   constructor() {
     super();
 
+    // state is set to an empty item array/0 points
     this.state = {
       items: [],
-      points: 0,
+      points: 0
     };
 
     // Uncomment this to spawn a single test item
-    //const testItem = this.spawnItem(Date.now());
-    //this.state.items.push(testItem);
+    const testItem = this.spawnItem(Date.now());
+    this.state.items.push(testItem);
 
     // Uncomment this to automatically spawn new items
     this.enableSpawner();
-
-    console.log(this.state);
   }
 
-  onItemClicked = () => {
+  onItemClicked = type => {
```

Comment:
You don't actually need to pass the type in here, you already do that check in `GameItem.js`.